tasklist/client/src/v2/api/index.ts

```diff
@@ -89,6 +92,56 @@ const api = {
       },
     );
   },
+  queryVariablesByUserTask: (
+    params: Pick<UserTask, 'userTaskKey'> & QueryVariablesByUserTaskRequestBody,
+  ) => {
+    const {userTaskKey, ...body} = params;
+    return new Request(
+      getFullURL(
+        tasklistEndpoints.queryVariablesByUserTask.getUrl({userTaskKey}),
+      ),
+      {
+        ...BASE_REQUEST_OPTIONS,
+        method: tasklistEndpoints.queryVariablesByUserTask.method,
+        body: Object.keys(body).length > 0 ? JSON.stringify(body) : undefined,
```

Comment:
I see this repeating a lot in the codebase, maybe we can extract it into a funciton.
`Object.keys(body).length > 0 ? JSON.stringify(body) : undefined`

Reply:
I removed it since it doesn't seem to be necessary it anymore